src/middlewares/userAuth.js

```diff
@@ -0,0 +1,35 @@
+const jwt = require("jsonwebtoken")
+const User = require("../models/User")
+const moment = require("moment-jalaali")
+
+exports.UserAuth = function name(req, res, next) {
+    // گرفتن توکن
+    try {
+        const token = req.headers.authorization
+        const data = jwt.verify(token, "123456")
```

Comment:
secret key should be defined in .env file and imporeted here. not directly inserted